com.unity.perception/Runtime/GroundTruth/LensDistortionCrossPipelinePass.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using Unity.Mathematics;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering;
+
+#if HDRP_PRESENT
+    using UnityEngine.Rendering.HighDefinition;
+#elif URP_PRESENT
+    using UnityEngine.Rendering.Universal;
+#endif
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    /// <summary>
+    /// Custom Pass which renders labeled images where each object labeled with a Labeling component is drawn with the
+    /// value specified by the given LabelingConfiguration.
+    /// </summary>
+    public class LensDistortionCrossPipelinePass : GroundTruthCrossPipelinePass
```

Comment:
out of date docs

Reply:
Also, the public API here will need to be documented

Reply:
Apart from updating this in the comment, where would I need to update the documentation?  Or is this more a comment that this text should be documenting the actual functionality of the pass (which is what I've addressed in an incoming change).

Reply:
Just that the text should reflect the feature. I don't necessarily think we need to add anything to the manual since this feature "just works" with no new UI or user control.

Reply:
Ok great - I did add change the text per that.  Doesn't look like github updated it here however.